src/otx/v2/adapters/torch/mmengine/mmseg/modules/models/utils/normalize.py

```diff
@@ -0,0 +1,66 @@
+"""Normalization."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from typing import Callable
+
+import torch
+import torch.nn.functional
+from torch import nn
+
+
+class OnnxLpNormalization(torch.autograd.Function):
+    """OnnxLpNormalization."""
+
+    @staticmethod
+    def forward(
+        ctx: Callable,  # noqa: ARG004
```

Comment:
Instead of ignoring the unused args and methods, the suggested way is to delete them at the beginning of the function. Something like

```python
    @staticmethod
    def forward(
        ctx: Callable,
        x: torch.Tensor,
        axis: int = 0,
        p: int = 2,
        eps: float = 1e-12,
    ) -> torch.Tensor:
        """Forward."""
        del ctx, p    # These args are not used.

        denom = x.norm(2, axis, True).clamp_min(eps).expand_as(x)
        return x / denom
```